accord-core/src/main/java/accord/coordinate/ExecuteTxn.java

```diff
@@ -192,11 +198,25 @@ protected Action process(Id from, ReadReply reply)
         CommitOrReadNack nack = (CommitOrReadNack) reply;
         switch (nack)
         {
-            default: throw new IllegalStateException();
+            default: throw new UnhandledEnum(nack);
+            case Waiting:
+                if (from.id != node.id().id)
+                    throw new UnhandledEnum(nack);
```

Comment:
nit: should we throw unhandled enum here, or maybe something more informative, like node id mismatch?

Reply:
It means we aren't expecting this reply from anyone but ourselves, and so we don't know how to handle the reply. I am happy to use another exception, but I don't think node id mismatch is correct, it's that the enum shouldn't have come from another node, and it did, so we don't know what to do.

Reply:
(That said, I could just `break` as it is totally fine to ignore this reply from another node really)

Reply:
I am mostly concerned that in case codebase diverges from the deployed version we might mistake this unhandled enum from the one on L201. I don't mind throwing here, just maybe assertion error, or some other exception? I don't mind swallowing the response in this case, either.